example.dev-config.yml

```diff
@@ -51,6 +51,9 @@ pulp_source_dir: "/home/vagrant/devel/pulpcore"
 # # Uncomment to deploy with apache instead of nginx
 # pulp_webserver_server: "apache"
 
+# # Comment to disable installer to enable repositories needed by Pulp
+pulp_repos_enabled: True
```

Comment:
As we discussed, I think this is better addressed with an "enable" variable for each of the 3 types of repos we add.